zubhub_frontend/zubhub/src/views/PageWrapper.jsx

```diff
@@ -512,14 +516,16 @@ function PageWrapper(props) {
                     onClose={e => handleSetState(handleProfileMenuClose(e))}
                   >
                     <MenuItem>
+                    <Tooltip title={props.auth.username} placement="top">
                       <Typography
                         variant="subtitle2"
                         color="textPrimary"
                         component="span"
                         className={classes.profileStyle}
                       >
-                        {props.auth.username}
+                        {props.auth.username.length<=14? props.auth.username : sliceLongName(props.auth.username) }
```

Comment:
Usually its nice to keep space between operators and other tokens for better readability, so this should be `length <= 14 ?`

Reply:
ok thanks for mentioning that i will make sure to add spaces